main/src/ca/uwaterloo/flix/language/ast/ops/SimplifiedAstOps.scala

```diff
@@ -318,7 +318,7 @@ object SimplifiedAstOps {
       //
       // NewChannel Expressions.
       //
-      case Expression.NewChannel(exp, tpe, loc) =>
+      case Expression.NewChannel(exp, _, tpe, loc) =>
```

Comment:
please add the variablename, in order to hold consistency with the rest of the file